spec/Block-Synchronization-Extension.md

```diff
@@ -19,13 +19,14 @@ Message :=
 ### Status
 
 ```
-Status(total_score, best_hash, genesis_hash)
+Status(nonce, best_hash, genesis_hash)
```

Comment:
How about `seq`?
`nonce` seems a random number not a monotonically increasing number.

Reply:
It seems better. Let me fix it.